### PR TITLE
Updated my(194h) keymap to make it more clean and useful

### PR DESCRIPTION
**Pull request by 194h**

### CODE REVIEW

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 1]
```diff
@@ -95,66 +127,23 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
-      TAP_ONCE(KC_4);
-      unregister_code(KC_RALT);
-    }
-    SEQ_ONE_KEY (KC_S) {
-      SEND_STRING("sudo !!");
-      TAP_ONCE(KC_ENT);
-    }
-  }
-}
+//TODO: Is there productivity gain in the use of unicode?
```
Comment (drashna) [inline keyboards/atreus62/keymaps/194h/keymap.c]: Well, not sure if it helps, but we did add a send string function for unicode, that you may want to check out: https://docs.qmk.fm/#/feature_unicode?id=send_unicode_hex_string

File: keyboards/atreus62/keymaps/194h/keymap.c [hunk 2]
```diff
@@ -95,66 +127,23 @@ const uint16_t PROGMEM keymaps[][MATRIX_ROWS][MATRIX_COLS] = {
...
+//	- Won't work in xterm.
+//	Create a layer for macros and/or unicode?
+//	Are macros useful?
```
Comment (drashna) [inline keyboards/atreus62/keymaps/194h/keymap.c]: You may want to check out my userspace. There are a number of useful macros, like `KC_MAKE` there.  And some trashy ones.

**Comment by drashna** [comment]
Looks good! 

Thanks!

File: keyboards/atreus62/keymaps/194h/keymap.c (hunk 1 above)
Comment (194h) [reply keyboards/atreus62/keymaps/194h/keymap.c]: It amazes me how fast you work and still have the time to help out in this way.
Thank you.
I'll definitely have a look at your suggestions.

Comment (drashna) [reply keyboards/atreus62/keymaps/194h/keymap.c]: Well, both are something that I enjoy doing. :) 

That and working from home makes it a lot easier. :)

**Pull request merged**